source/atlas-cli-changelog.txt

```diff
@@ -12,6 +12,21 @@
    :depth: 1
    :class: singlecol
 
+.. _atlas-cli-1.5.0:
+
+{+atlas-cli+} 1.5.0
+-------------------
+
+Released 9 February 2023
+
+- Updates the :ref:`atlas-config-describe` command to no longer redact 
+  public |api| keys.
+- Updates the default output format for the :ref:`atlas-customDbRoles-list` 
+  command to include inherited roles.
+- Deprecates the :ref:`atlas-integrations-create-NEW_RELIC` command.
+- Adds support for cluster termination protection.
```

Comment:
```suggestion
- Adds support for cluster termination protection. You can enable
  cluster termination protection when you run
  :ref:`atlas-clusters-create` or :ref:`atlas-clusters-update`
  with the --enableTerminationProtection option and you can disable cluster 
  termination protection when you run :ref:`atlas-clusters-update` with
  the --disableTerminationProtection flag.
```

Reply:
Decided to format these as list items to reduce clutter. Let me know what you think!